internal/cmd/config.go

```diff
@@ -247,6 +247,11 @@ func (config *Config) DenyDir() string {
 	return filepath.Join(config.DataDir, "deny")
 }
 
+// RequiredDir is the folder where all the "required" files are stored.
+func (config *Config) RequiredDir() string {
```

Comment:
I find the naming of this directory confusing. It does not contain "required" files but _required files that have been approved_. Maybe `allowedRequired` and `AllowedRequiredDir()` would be more fitting?

This also applies to all other mentions of "required".

Reply:
[`8145cf7` (#1530)](https://github.com/direnv/direnv/pull/1530/commits/8145cf7ddc3701010d42bcd8c1a80305eccc5a98)